customisation/language-identification.md

```diff
@@ -0,0 +1,16 @@
+---
+grand_parent: Building and research
+parent: Customisation
+title: Language identification
+description: Identifying the language of a text
+---
+
+**Language identification** tools identify the language of an unknown text.
+In machine translation, they are used in building monolingual [training data](/customisation/training-data.md) from large collections of unidentified texts such as webpages.
+They are also used to automatically identify the source language in many machine translation systems when the source language is unspecified.
```

Comment:
Shall we mention something like...?

"Some language identification methods are n-gram probability, dictionaries, and more."

I fear this may end up not being evergreen (or is even not evergreen now), but we can fix it by saying "By 2022, ...".

Reply:
Yeah I couldn't decide either way, but if you think it'd be helpful I'm happy to add that.

Reply:
I think it can be interesting to mention it as long as it adds info that's accessible to all our readers profile.
Perhaps `n-gram probability` can be tricky, but if we link to the `n-gram` article, we create a nice reading path for the audience. 
What do you think?